src/ts/uifactory.ts

```diff
@@ -41,7 +41,6 @@ import { MetadataLabel, MetadataLabelContent } from './components/metadatalabel'
 import { PlayerUtils } from './playerutils';
 import { Label } from './components/label';
 import { CastUIContainer } from './components/castuicontainer';
-import { VolumeControlButton } from './components/volumecontrolbutton';
```

Comment:
@protyposis can we remove the `VolumeControlButton` at all?

Reply:
Let's keep it, maybe somebody can make use of it.